packages/services/api/src/modules/preflight-script/module.graphql.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  type PreflightScript {
+    id: ID!
+    sourceCode: String!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    # TODO: should I add this field?
+    # createdBy: User!
```

Comment:
I think we should expose it. In our UI we could show various scripts available in the org for user to pick from. Like I can create a script and tell you go find the script I made

Reply:
cc @n1ru4l wdyt